compatibility_server/pip_checker.py

```diff
@@ -354,46 +355,79 @@ def _build_command(self, subcommands: List[str]):
         """Build pip commands."""
         return self._pip_command + subcommands
 
+    def _clone_repo(self, container: docker.models.containers.Container):
+        """Shallow clone the google-cloud-python repository."""
```

Comment:
It would be very easy to make this generic, so why not do that?

Reply:
Done.